ui/packages/security-servlet-logout/src/main/webapp/js/main.js

```diff
@@ -25,42 +25,8 @@
 
     if (actions.length !== 0) {
       $('#noActions').toggleClass('is-hidden', actions.length !== 0)
-      $('#actions').toggleClass('is-hidden', actions.length === 0)
       actions.forEach(function(action) {
-        if ($.url().param('noPrompt')) {
-          doLogout(action)
-        } else {
-          var $row = $('<tr></tr>')
-          var $realm = $('<td></td>')
-          $realm.html(action.realm)
-          var $realmDescription = $(
-            '<a href="#" class="description" tabindex="-1"></a>'
-          )
-          $realmDescription.data('title', action.description)
-          $realmDescription.data('placement', 'right')
-          $realmDescription.append(
-            $('<i class="glyphicon glyphicon-question-sign"></i>')
-          )
-          $realm.append($realmDescription)
-          var $user = $('<td></td>')
-          $user.html(action.auth)
-          var $logout = $('<td></td>')
-          var $button = $('<button>')
-            .attr('id', 'logoutButton')
-            .text('Logout')
-            .addClass('btn btn-primary float-right')
-            .click(function() {
-              doLogout(action)
-            })
-          $logout.append($button)
-          $row
-            .append($realm)
-            .append($user)
-            .append($logout)
-          $('#actions')
-            .find('tbody')
-            .append($row)
-        }
+        doLogout(action)
```

Comment:
❓ If we have multiple actions, wont this just set the `window.location` to the last action's url?

Reply:
Correct. This is doing what we used to do when we had no-prompt passed in https://github.com/codice/ddf/blob/master/ui/packages/security-servlet-logout/src/main/webapp/js/main.js#L30-L31